<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QWidget" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>431</width>
     <height>681</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>户外空气质量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>250</pointsize>
     </font>
    </property>
    <property name="text">
     <string>25</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>580</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AirQuality</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TempAndHumi">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>840</y>
     <width>721</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#TempAndHumi{
border-radius:8px;
background:#FFFFFF;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TempP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>HumiP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>25/26°C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>室内/外</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>25/26%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>室内/外</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>适宜活动</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>适宜活动</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>580</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>适宜活动</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="WeatherPage">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>40</y>
     <width>901</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#WeatherPage{
border-radius:8px;
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_16">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>16</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_17">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>16</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_20">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_21">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_22">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_23">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_18">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>16</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_19">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>16</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_24">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>雨天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>14：00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>雨天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>14：00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>雨天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>14：00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>14：00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>雨天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>14：00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>雨天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>330</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>No2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>500</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>330</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>500</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>330</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Pm2.5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_13">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>500</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_14">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>330</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>lux</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_15">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>500</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="CurtainsPage">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>840</y>
     <width>742</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#CurtainsPage{
border-radius:8px;
/*background-color:transparent;*/
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="AllCurtains_Open">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:8px;
background:#FFFFFF;
/*border:1px solid black;</string>
     </property>
     <property name="text">
      <string>开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AllCurtains_Stop">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:8px;
background:#FFFFFF;
/*border:1px solid black;</string>
     </property>
     <property name="text">
      <string>停</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AllCurtains_Close">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:8px;
background:#FFFFFF;
/*border:1px solid black;</string>
     </property>
     <property name="text">
      <string>关</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="WindowBox">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>600</y>
     <width>61</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
background:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CurtainsBox">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>840</y>
     <width>61</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
background:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QFrame" name="WindowPage">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>600</y>
     <width>741</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:8px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="AllWindow_Stop_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:8px;
background:#FFFFFF;
/*border:1px solid black;</string>
     </property>
     <property name="text">
      <string>停</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AllWindow_Close_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:8px;
background:#FFFFFF;
/*border:1px solid black;</string>
     </property>
     <property name="text">
      <string>关</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AllWindow_Open_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:8px;
background:#FFFFFF;
/*border:1px solid black;</string>
     </property>
     <property name="text">
      <string>开</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
